packages/notifi-react-card/lib/components/subscription/subscription-card-views/preview-panel/UserInfoPanel.tsx

```diff
@@ -17,20 +17,46 @@ export type UserInfoPanelProps = {
     telegram?: UserInfoSection;
     sms?: UserInfoSection;
     EditButton: string;
+    confirmation: {
+      email: string;
+      emailLabel: string;
+      telegram: string;
+      telegramLabel: string;
+    };
   }>;
   data: CardConfigItemV1;
+  confirmationLabels?: {
+    email: string;
+    telegram: string;
+  };
 };
 export const UserInfoPanel: React.FC<UserInfoPanelProps> = ({
   classNames,
   data,
+  confirmationLabels = {
+    email: 'Resend link',
+    telegram: 'Verify Id',
+  },
```

Comment:
let's put the defaults in-line (i.e. on line 95 it would be `confirmationLabels?.email ?? 'Resend link'`